homeassistant/components/energy/validate.py

```diff
@@ -2,20 +2,31 @@
 from __future__ import annotations
 
 import dataclasses
-from typing import Any
+from typing import Any, Sequence
 
 from homeassistant.components import recorder, sensor
 from homeassistant.const import (
     ENERGY_KILO_WATT_HOUR,
     ENERGY_WATT_HOUR,
     STATE_UNAVAILABLE,
     STATE_UNKNOWN,
+    VOLUME_CUBIC_FEET,
+    VOLUME_CUBIC_METERS,
 )
 from homeassistant.core import HomeAssistant, callback, valid_entity_id
 
 from . import data
 from .const import DOMAIN
 
+ENERGY_USAGE_UNITS = (ENERGY_KILO_WATT_HOUR, ENERGY_WATT_HOUR)
+ENERGY_UNIT_ERROR = "entity_unexpected_unit_energy"
+GAS_USAGE_UNITS = (
```

Comment:
`ENERGY_KILO_WATT_HOUR` is currently added to `GAS_USAGE_UNITS` for backwards compatibility?

Reply:
Forward compatibility. We've gotten feedback that some users track gas usage in kWh and so we're planning on supporting that in the next release.

Reply:
Gas in kWh. Okay you learn everyday something new 😃

But then should `Wh` not also be added to be complete?